checkers/output_tolerance_check.py

```diff
@@ -66,13 +68,17 @@ def check():
         # check for output lists
         try:
             nout_list = get_param(rundir+nlfile, lnout)
+            assert(nout_list != '')
```

Comment:
thanks to the assert, I saw this is in fact not working in some tests, see
https://github.com/C2SM-RCM/testsuite/issues/37